src/Voron/Schema/Updates/From23.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.IO;
+using Voron.Impl.FileHeaders;
+
+namespace Voron.Schema.Updates;
+
+public class From23  : IVoronSchemaUpdate
+{
+    public bool Update(int currentVersion, StorageEnvironmentOptions options, HeaderAccessor headerAccessor, out int versionAfterUpgrade)
+    {
+        foreach (var unusedFile in Directory.GetFiles(options.JournalPath.FullPath, "recyclable-journal.*"))
+        {
+            File.Delete(unusedFile);
```

Comment:
IOExtensions.Delete?

Reply:
This is in `Voron`, that is in `Raven.Server`

Reply:
I did added exception handling to ignore failures.